src/components/wallet/earn/Validate/ValidatorInfo.vue

```diff
@@ -0,0 +1,207 @@
+<template>
+    <div class="validator_card">
+        <Spinner v-if="loading" style="color: var(--primary-color)"></Spinner>
+
+        <div v-if="!loading" class="validator_child_card">
+            <div class="validator_info">
+                <div class="alt_validator_info">
+                    <div>
+                        <Tooltip style="display: inline-block" text="Staking Start Date">
+                            <v-icon class="icon-mdi-camino">mdi-calendar-blank</v-icon>
+                            <label>{{ startTime }}</label>
+                        </Tooltip>
+                    </div>
+                    <div>
+                        <Tooltip style="display: inline-block" text="Staking End Date">
+                            <v-icon class="icon-mdi-camino">mdi-calendar-remove-outline</v-icon>
+                            <label>{{ endTime }}</label>
+                        </Tooltip>
+                    </div>
+                    <div>
+                        <Tooltip style="display: inline-block" text="Up Time">
+                            <v-icon class="icon-mdi-camino">mdi-arrow-up-bold</v-icon>
+                            <label>{{ upTime }} %</label>
+                        </Tooltip>
+                    </div>
+                    <div>
+                        <Tooltip style="display: inline-block" text="Remaining Validation Period">
+                            <v-icon class="icon-mdi-camino">mdi-clock-time-five-outline</v-icon>
+                            <label>{{ reaminingValidation }}</label>
+                        </Tooltip>
+                    </div>
+                </div>
+            </div>
+            <br />
+            <div>
+                <h4 class="input_label">{{ $t('earn.validate.nodeId') }}</h4>
+                <span class="disabled_input" role="textbox">
+                    {{ nodeId }}
+                </span>
+            </div>
+            <div>
+                <h4 class="input_label">Deposit Amount</h4>
```

Comment:
Same here, please use translations. Just create a new key in `en.json` and use it here and in other places you added